lockrenewal.go

```diff
@@ -0,0 +1,77 @@
+package servicebus
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-amqp-common-go/log"
+	"github.com/Azure/azure-amqp-common-go/rpc"
+	otlogger "github.com/opentracing/opentracing-go/log"
+	"time"
+
+	"pack.ag/amqp"
+)
+
+const (
+	serviceBuslockRenewalOperationName = "com.microsoft:renew-lock"
```

Comment:
I'm not sure exactly how I'd like to organize these constants, but I too have been finding a lot of need for this. Maybe we could spin this and other const definitions into an `operations.go` file that just collects operation names and other related constants?

Reply:
Sounds like a good plan I'll create the file and relocate these.